src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
```

Comment:
Avoid beginning method implementation with an empty line. Please remove the empty line at the start of the constructor.